code/game/world.dm

```diff
@@ -67,7 +70,9 @@ GLOBAL_VAR(restart_counter)
 #ifdef USE_BYOND_TRACY
 #warn USE_BYOND_TRACY is enabled
 	if(!tracy_initialized)
-		init_byond_tracy()
+		GLOB.tracy_log = init_byond_tracy()
+		if(GLOB.tracy_log)
+			SEND_TEXT(world.log, "Tracy profile log located at [GLOB.tracy_log]")
```

Comment:
Logger isn't initialized at this point, so we have to send to `world.log` directly

Reply:
Anything unnecessary to this function should not be present. Log this somewhere else, i.e. /world/New()